rfc-013-use-docker-to-deploy-and-run-applications-in-containers.md

```diff
@@ -0,0 +1,76 @@
+# Use Docker to deploy and run our applications in containers
+
+## Summary
+
+All of the applications we build are tested and run using Docker containers.
```

Comment:
I'm not sure how I feel about "all" here. It won't always be possible, given our clients' preferences (e.g. Heroku for Hackney). Perhaps this could be improved by calling out that it should be our strong preference, but we are permitted to diverge based on client needs?

Reply:
I now see that we say we can override with an ADR, but maybe it would be worth including that in the summary?